### PR TITLE
Added more inline phpdoc tests

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

**Comment by keradus** [comment]
:+1:

File: Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php [hunk 1]
```diff
@@ -272,4 +272,35 @@ public function testWrongCasedPhpdocTagIsNotAltered()
...
+    public function testInlineDoc()
+    {
+        $expected = <<<'EOF'
+<?php
+    /**
+     * Does stuffs with stuffs.
+     *
+     * @param array $stuffs {
+     *     @type bool $foo
+     *     @type int $bar
```
Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php]: will these be aligned by the `PhpdocAlignFixer` ?

Comment (GrahamCampbell) [inline Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php]: Yes, but that's unrelated to this PR?

Comment (keradus) [inline Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php]: I think that @SpacePossum asked for such a case, as you are extending them

Comment (GrahamCampbell) [inline Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php]: Right. Can we leave that to another PR, since I have nothing to do with the alignment fixer. TBH, I'd still like to see that rewritten to use the phpdoc classes we have for consistent behaviour.

Comment (keradus) [inline Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php]: Would adding that one tests, that you told would pass, be a problem?

> TBH, I'd still like to see that rewritten to use the phpdoc classes we have for consistent behaviour.

Go for it! :)

Comment (GrahamCampbell) [inline Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php]: > Go for it! :)

Ha. If I had the time, I'd love to. It's not a 5 min job I'm afraid. ;)

Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php]: it just got curious about it, that is all, now I know it will be done by the other fixer, so

>Can we leave that to another PR,

sure :) 
thanks for clearing it up :+1:

**Comment by keradus** [comment]
Thank you @GrahamCampbell.
